src/config.yml

```diff
@@ -0,0 +1,35 @@
+# VOA Swahili
+SOURCES:
+  - https://www.voaswahili.com/z/4658/?p={}
+  - https://www.voaswahili.com/z/4657/?p={}
+  - https://www.voaswahili.com/z/4660/?p={}
+  - https://www.voaswahili.com/z/4768/?p={}
+  - https://www.voaswahili.com/z/4659/?p={}
+  - https://www.voaswahili.com/z/2774/?p={}
+  - https://www.voaswahili.com/z/2775/?p={}
+  - https://www.voaswahili.com/z/2783/?p={}
```

Comment:
I am particularly curious as to how we get this sources and the page limits too
If say I decide to scrape for a new language; how will this work

@theyorubayesian do you have any idea?

Reply:
This was done manually, on your first click on a category, let's say **world** category, it does not initiate pagination yet until you click on _read more_, you can the edit the _url_ to check for the page limit

Reply:
No ideas specifically. For BBC, I visited all the pages on the homepage and searched for links matching a particular pattern. See my [get_topics](https://github.com/theyorubayesian/bbc_pidgin_scraper/blob/8461ebc8b9a8e40e8c44824cd3dd18fe1969e960/scraper.py#L205-L221) function

Reply:
Hmmm; I was thinking if there was a way to get all the pages in an automated way since the idea is to scrape everything anyway